components/articles-inputs/parameter/StateParameter.tsx

```diff
@@ -0,0 +1,44 @@
+import { getIn } from "immutable";
```

Comment:
Ce fichier utilise les recommandations de react-redux sur l'intégration de TypeScript : https://redux.js.org/recipes/usage-with-typescript#typing-the-connect-higher-order-component

Reply:
La librairie `immutable` est très utilisée avec `redux` et permet facilement de changer une valeur de l'état en créant clonant des objets lorsque cela est nécessaire. Ici `getIn` est juste utilisé pour lire une propriété à partir d'un chemin.